### ISSUE TITLE
Allow (Easy) Themeing

### ISSUE THREAD

**Issue by DaSourcerer** [hosannahighertech/yii2-yiiboard#3]
Much like #2: Providing a basic bootstrapped theme as suggested in #1 would be a great start. We still need a way to let users install and choose a theme. Perhaps we can handle them like a special type of plugin?

**Comment by mtangoo** [comment]
Yeah. Having plugin is really fine (core plugin). Since Yii2 have [much done for us](https://github.com/yiisoft/yii2/blob/master/docs/guide/output-theming.md) we will just need to know few information about a theme to load it.

Additionaly we might need theme installer (Like mybb) which I suggest be a simple theme folder with single XML with theme information (to add to admin Panel). We can then hash theme folder depending on generated theme ID (by installer of course) to avoid folder name colision
